src/poker/GameResult.java

```diff
@@ -27,4 +28,9 @@ public void increasePlayer2Wins(int wynik){
     public void increaseTies(int wynik){
         this.ties += wynik;
     }
+    public enum result {
+        WIN_PLAYER1,
+        WIN_PLAYER2,
+        DRAW
+    }
```

Comment:
Ogolnie spoko ze zrobiles ALE
nazwa enuma zawsze z duzej i jak juz wkladasz to do srodka jakies klasy to powinno byc to gdzies na gorze zeby odrazu sie w oczy rzucalo i nazwa result jest dosc slaba bo malo inforamcji dostarcza jak jej uzyć polecam zmienic na np HandWinner

Reply:
ZMIEN TA NAZWE

Reply:
Po peirwsze nazwa winningHand jest slaba bo sie odnosi bardziej jaki hand wygral niz jaki player,
PlayerWinner jest lepsza, I CI PISALEM ZE PIERWSZA LITERA W NAZWIE ENUMA MA BYC Z DUEJ

Reply:
KUR** NO 4 komentarz na to robie